vite.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import checker from "vite-plugin-checker";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  // Make vite run on port 3000.
+  server: {
+    port: 3000,
```

Comment:
We should run this on another port so that there aren't constant collisions with Spruce

Reply:
ah so true 🙏 the default is `5173`,  I'm pretty sure it's supposed to somewhat resemble the word VITE lol

I can change it if there's a number you had in mind!